src/components/Button/Button.protractor.driver.ts

```diff
@@ -1 +1,17 @@
-export {buttonDriverFactory, ButtonDriver} from 'wix-ui-core/dist/src/components/Button/Button.protractor.driver';
+import { ElementFinder } from 'protractor';
+import { DriverFactory } from 'wix-ui-core/dist/src/common/BaseDriver.protractor';
+import {buttonDriverFactory as coreButtonDriverFactory, ButtonDriver as CoreButtonDriver} from 'wix-ui-core/dist/src/components/Button/Button.protractor.driver';
+import {mouseEnter, mouseLeave} from 'wix-ui-test-utils/protractor';
+
+export interface ButtonDriver extends CoreButtonDriver {
+  mouseEnter: () => Promise<void>;
+  mouseLeave: () => Promise<void>;
+}
+
+export const buttonDriverFactory: DriverFactory<ButtonDriver> = (element: ElementFinder): ButtonDriver => {
+  return {
+    ...coreButtonDriverFactory(element),
+    mouseEnter: async () => mouseEnter(element),
```

Comment:
should it actually be a driver function? why should the user do it? in his code? if it's only for our test, it can stay internal